packages/applications-service-api/src/controllers/submissions.js

```diff
@@ -10,11 +11,21 @@ const createSubmissionController = async (req, res) => {
 };
 
 const completeSubmissionController = async (req, res) => {
-	await completeSubmission({
+	const submissionDetails = {
 		submissionId: req.params.submissionId,
 		caseReference: req.body?.caseReference,
 		email: req.body?.email
-	});
+	};
+
+	if (
```

Comment:
If there is no body or empty body then I think the existing Front end call for NI integration would get past this check and work.  Is that by design?

I think we may need a front end change to send the body but perhaps that is not for this ticket.

Reply:
Yeah, that's intentional. The existing NI call has no body so I've made email and caseref optional (but if you provide one, then you must provide the other)